diff -urN openwebmail_org/cgi-bin/openwebmail/etc/openwebmail.conf openwebmail_mod/cgi-bin/openwebmail/etc/openwebmail.conf
--- openwebmail_org/cgi-bin/openwebmail/etc/openwebmail.conf	2004-01-03 05:16:50.000000000 +0100
+++ openwebmail_mod/cgi-bin/openwebmail/etc/openwebmail.conf	2004-02-19 12:28:22.000000000 +0100
@@ -11,9 +11,9 @@
 domainnames		auto
 auth_module		auth_unix.pl
 mailspooldir		/var/mail
-ow_cgidir		/usr/local/www/cgi-bin/openwebmail
+ow_cgidir		/home/services/httpd/cgi-bin/openwebmail
 ow_cgiurl		/cgi-bin/openwebmail
-ow_htmldir		/usr/local/www/data/openwebmail
+ow_htmldir		/home/services/httpd/html/openwebmail
 ow_htmlurl		/openwebmail
 logfile			/var/log/openwebmail.log
 
