diff -urN openwebmail_org/cgi-bin/openwebmail/auth/auth_unix.pl openwebmail_mod/cgi-bin/openwebmail/auth/auth_unix.pl
--- openwebmail_org/cgi-bin/openwebmail/auth/auth_unix.pl	2003-12-07 03:23:20.000000000 +0100
+++ openwebmail_mod/cgi-bin/openwebmail/auth/auth_unix.pl	2004-02-19 12:46:45.000000000 +0100
@@ -21,7 +21,7 @@
 }
 
 my $passwdfile_plaintext = $conf{'passwdfile_plaintext'} || '/etc/passwd';
-my $passwdfile_encrypted = $conf{'passwdfile_encrypted'} || '/etc/master.passwd';
+my $passwdfile_encrypted = $conf{'passwdfile_encrypted'} || '/etc/shadow';
 my $passwdmkdb = $conf{'passwdmkdb'} || '/usr/sbin/pwd_mkdb';
 
 my $check_nologin = $conf{'check_nologin'} || 'no';
